config.py

```diff
@@ -14,4 +14,6 @@
 
 CWL_VERSION = 'v1.0'
 
+# Location of nanopub tool
+ROOT_DIR = Path(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
For now I've only moved NANOPUB_SCRIPT out of config.py, since that's the only thing the python lib actually needs. config.py is not included in the python package otherwise, since the CLI stuff etc is not currently in there. This should all get resolved with Backlog item 57.